src/AskMeAnything/__test__/index.test.ts

```
@@ -0,0 +1,23 @@
+  log: jest.fn(),
+  executionContext: { functionName: jest.fn() }
+} as unknown as InvocationContext;
+
+export const buildHttpRequest = (): HttpRequest => ({
+  headers: '',
+  method: 'GET',
+  params: undefined,
+  query: {},
+  url: "",
+  user: undefined,
+}) as unknown as HttpRequest;
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)